pkg/drivers/hyperv/hyperv_windows.go

```diff
@@ -304,6 +305,11 @@ func (d *Driver) Stop() error {
 
 // Remove removes an host
 func (d *Driver) Remove() error {
+	if _, stderr, err := powershell.Execute(`Hyper-V\Get-VM`, d.MachineName); err != nil &&
+		strings.Contains(stderr, "Hyper-V was unable to find a virtual machine") {
```

Comment:
Can this be localized? Not clear from a quick look at  the powershell code if the locale is always going to be an English one.

Reply:
yes not really good. What if I dont check the error message?
If the driver can't get the VM, then Remove returns nil directly.

Reply:
done!